### PR TITLE
Initial bundle visualization setup

### PR DESCRIPTION
**Pull request by eldiegod**
#792

### CODE REVIEW

File: config-overrides.js [hunk 1]
```diff
@@ -14,6 +15,13 @@ module.exports = {
   webpack: override(
     addBabelPlugin('@emotion/babel-plugin'),
     addWebpackPlugin(new StylelintPlugin({ files: './src/**/*.{tsx,ts}' })),
+    addWebpackPlugin(
+      new BundleAnalyzerPlugin({
+        // use value from environment var STATS or 'disabled'
+        analyzerMode: process.env.STATS || 'disabled',
```
Comment (kdembler) [inline config-overrides.js]: What would possible values for `STATS` be?

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "url": "https://github.com/Joystream/joystream/issues"
   },
   "scripts": {
+    "bundle-stats": "STATS=server npm run build",
```
Comment (kdembler) [inline package.json:19]: Does stats generation take some considerable time? I'm wondering whether maybe we should just enable it by default with a build?

Comment (eldiegod) [reply package.json:19]: I don't think it adds much more time but this opens a server and a tab on the browser, the build does make a file by default called stats.json which can be used as input on https://chrisbateman.github.io/webpack-visualizer/
or to open the the bundle analizer from the cli

File: config-overrides.js (hunk 1 above)
Comment (eldiegod) [reply config-overrides.js]: These
> Default: server. In server mode analyzer will start HTTP server to show bundle report. In static mode single HTML file with bundle report will be generated. In json mode single JSON file with bundle report will be generated. In disabled mode you can use this plugin to just generate Webpack Stats JSON file by setting generateStatsFile to true.

**Review by kdembler — APPROVED** [review]

**Pull request merged**